### PR TITLE
Mission1 art gallery game

### PR DESCRIPTION
**Pull request by mariana-perezh**
We still need to actually make the gallery and add the images (and the blurring) but this is the mvp for now

### CODE REVIEW

**Comment by mariana-perezh** [comment]
Won't merge for now but if anyone wants to make a gallery in the meantime please feel free

**Review by Wixx19 — APPROVED** [review]
Good to go!

**Comment by mariana-perezh** [comment]
Added more commits after approval, kind of screwed it up. Check new branch 'goingBack'

**Pull request closed without merging**
